wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos/kotlin/DeprecatedEnum.kt

```diff
@@ -5,23 +5,25 @@ package com.squareup.wire.protos.kotlin
 import com.squareup.wire.EnumAdapter
 import com.squareup.wire.ProtoAdapter
 import com.squareup.wire.WireEnum
+import kotlin.Boolean
 import kotlin.Deprecated
 import kotlin.Int
 import kotlin.jvm.JvmField
 import kotlin.jvm.JvmStatic
 
 enum class DeprecatedEnum(
-  override val value: Int
+  override val value: Int,
+  val deprecated: Boolean?
```

Comment:
@oldergod was this what you were seeing for Java as well? Feels like a bug. @swankjesse do you know why we're printing out `deprecated` as an enum field in addition to `@Deprecated`?

Reply:
Java does that indeed. https://github.com/square/wire/pull/1315/files#diff-e253bdd1b1acf18fd5cdeb67a950ecc8R526
Renee and I talked and we decided to first do as what Java is already doing. If that's a bug, let's fix it later in one go.

Reply:
SGTM!

Reply:
#1312